src/components/SegmentedTopBar.js

```diff
@@ -39,60 +40,54 @@ const width = Dimensions.get("screen").width - 32;
  * A Segment Picker component that provides toggling between two or more views.
  *
  * <div class="screenshots">
- *   <img src="screenshots/segmentPicker/segmentpickers.png" />
+ *   <img src="screenshots/segmentedTopBar/segmentedTopBars.png" />
  * </div>
  *
  *   ## Usage
  * ```js
  * import * as React from 'react';
- * import { Container, SegmentPicker } from '@bigbinary/neetoui-rn';
+ * import { SegmentedTopBar } from '@bigbinary/neetoui-rn';
  *
  * export default function Main(){
- *   const [tabIndex, setTabIndex] = useState(1);
  *
  *  return (
- *    <Container flex={1} alignItems="center" justifyContent="center">
- *      <Container my={3}>
- *        <SegmentPicker
- *          tabs={["On", "Off"]}
- *          currentIndex={tabIndex}
- *          onChange={setTabIndex}
+ *    <NavigationContainer>
```

Comment:
@injas-bigbinary import `NavigationContainer`,  creattion of `Tab` is missing in the example.